### ISSUE TITLE
Qualifier Instantiation Error 

### ISSUE THREAD

**Issue by ranjitjhala** [ucsd-progsys/liquid-fixpoint#119]
See `tests/todo/len00.fq` which works with `fixpoint` but not `fixpoint -n`. This also is the short test demonstrating the problem here:

https://github.com/ucsd-progsys/liquid-fixpoint/pull/118

**Comment by ranjitjhala** [comment]
somehow the qual-inst env **does not** have the `constant` in them, c.f.

```
Trace: [okInst [p := PAtom Ge (EApp "len" defined from: "tests/todo/len00.fq" (line 4, column 28) to: "tests/todo/len00.fq" (line 4, column 31) [EVar "v"]) (ECon (I 0)) ]] : Just Unbound Symbol len
 Perhaps you meant: v
```

**Comment by ranjitjhala** [comment]
Fixed by c815e05f40b15bf5db0eb43a926a5beb6de3714e which also (finally solves #112 ) i.e. `vector2.hs.fq` (however `vecloop.hs.fq` remains open.)
